app/views/stories/show.html.erb

```diff
@@ -60,15 +59,15 @@
           <% end %>
 
           <% if can? :create, Task.new %>
-            <%= simple_form_for [@story.project, @story, Task.new], html: { class: "pure-form pure-form-stacked" } do |f| %>
+            <%= simple_form_for [@project, @story, Task.new], html: { class: "pure-form pure-form-stacked" } do |f| %>
               <%= f.input :label, label: false, placeholder: "Add a new task...", input_html: { autocomplete: 'off', class: "atwho" } %>
             <% end %>
           <% end %>
         </div>
         <div class="attachments">
-          <% if @story.attachments.any? %>
+          <% if @attachments.size > 0 %>
```

Comment:
Lot of changing from `.any?` `.empty?` or `.count` to `.size` this is because we already have the collections loaded into memory in the controller, so no point doing an extra query to see if we have any or not